src/nginx/azext_nginx/aaz/latest/nginx/deployment/_update.py

```diff
@@ -0,0 +1,697 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "nginx deployment update",
+)
+class Update(AAZCommand):
+    """Update an Nginx deployment
+
+    :example: Deployment Update
+        az nginx deployment update --name myDeployment --resource-group myResourceGroup --location eastus2 --tags tag1="value1" tag2="value2" --enable-diagnostics true
```

Comment:
It's not required to add true for boolean argument

```suggestion
        az nginx deployment update --name myDeployment --resource-group myResourceGroup --location eastus2 --tags tag1="value1" tag2="value2" --enable-diagnostics
```